src/tools/schema2kotlin.ts

```diff
@@ -257,6 +268,21 @@ ${this.opts.wasm ? `
 
 class ${name}_Spec() : ${this.getType('EntitySpec')}<${name}> {
 
+    companion object {
+        val schema = Schema(
+            listOf(${schemaNames.join('\n                ')}),
+            SchemaFields(
+                singletons = mapOf(
+                    ${this.singletonSchemaFields.join(',\n                    ')}
+                ),
+                collections = mapOf(
+                    ${this.collectionSchemaFields.join(',\n                    ')}
+                )
+            ),
+            "${schemaHash}"
+        ) 
+    }
+    
     override fun create() = ${name}()
     ${!this.opts.wasm ? `
     override fun deserialize(data: RawEntity): ${name} {
```

Comment:
Can you re-generate and add the golden files to help with the review?

bazel test src/tools/tests/...
and follow the test failure log instructions

Reply:
I definitely will! I'm still playing around with show I want users to access the schemas.

Reply:
Generated!